Vagrantfile

```diff
@@ -22,6 +23,12 @@ if options['github_token'].nil? || options['github_token'].to_s.length != 40
   exit
 end
 
+# check host ip
+if options['ip'] == "1"
```

Comment:
I'm afraid you are confusing terms here. My understanding of host and guest are:

- host: the machine that runs vagrant
- guest: the virtual machine

host IP is something like `192.168.x.1`
guest IP is configurable.

Also see my comments below, I think this is confusing.
Maybe it can be solved by changing the naming:

`netmask` -> `ip_prefix`
`ip` could stay as is then.